recipe/meta.yaml

```
@@ -28,11 +31,19 @@ requirements:
     - libpostal
     - six
 test:
```

```suggestion

test:
```

one space between sections